CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 1.7.1
+* Ensure `data` is not nil when logging wrong #exception usage.
```

Comment:
[super nitpic] Should remove the period at the end? 👾

Reply:
👌